calcom/cal.com
# fix: added loading state to button

# fix : add loading state button in logout page for a good UX

## What does this PR do?

This PR adds a loading state to the button which moves the user login page.



Fixes #11113



https://www.loom.com/share/4b5b44da1aac457aa588fe826e3dbed9?sid=30bca723-beb6-4017-a05e-a4d6477e2d03

## Requirement/Documentation







- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?

Please check the video and the code changes, that must be enough for testing.



- Are there environment variables that should be set? No
- What are the minimal test data to have? None
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR 

## Mandatory Tasks


## Checklist








- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
> Ran the tests, there are some tests which are failed but not related the changes made by me
